src/Microsoft.AspNet.Server.Kestrel/Http/FrameRequestHeaders.cs

```diff
@@ -21,11 +21,11 @@ public Enumerator GetEnumerator()
 
         public partial struct Enumerator : IEnumerator<KeyValuePair<string, StringValues>>
         {
-            private FrameRequestHeaders _collection;
-            private long _bits;
+            private readonly FrameRequestHeaders _collection;
+            private readonly long _bits;
             private int _state;
             private KeyValuePair<string, StringValues> _current;
-            private bool _hasUnknown;
+            private readonly bool _hasUnknown;
             private Dictionary<string, StringValues>.Enumerator _unknownEnumerator;
```

Comment:
Trying to find specific for enum; but here is example of don't make mutable struct readonly: https://github.com/dotnet/coreclr/blob/bc146608854d1db9cdbcc0b08029a87754e12b49/src/mscorlib/src/System/Runtime/CompilerServices/AsyncMethodBuilder.cs#L40